src/Dapr.AspNetCore/TopicAttribute.cs

```diff
@@ -105,6 +105,28 @@ public TopicAttribute(string pubsubName, string name, bool enableRawPayload, str
             this.MetadataSeparator = metadataSeparator;
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TopicAttribute" /> class.
+        /// </summary>
+        /// <param name="pubsubName">The name of the pubsub component to use.</param>
+        /// <param name="name">The topic name.</param>
+        /// <param name="deadLetterTopic">The dead letter topic name.</param>
+        /// <param name="enableRawPayload">The enable/disable raw pay load flag.</param>
+        /// <param name="ownedMetadatas">The topic owned metadata ids.</param>
+        /// <param name="metadataSeparator">Separator to use for metadata.</param>
+        public TopicAttribute(string pubsubName, string name, string deadLetterTopic, bool enableRawPayload, string[] ownedMetadatas = null, string metadataSeparator = null)
```

Comment:
Were we not able to make a constructor with `TopicOptions`?

Reply:
Discussed offline